rules/use-t-well.js

```diff
@@ -3,35 +3,91 @@ const {visitIf} = require('enhance-visitors');
 const util = require('../util');
 const createAvaRule = require('../create-ava-rule');
 
-const isMethod = name => util.executionMethods.has(name);
+class MicroCorrecter {
+	constructor(words) {
+		this.words = new Set(words);
+
+		const letters = new Set();
+		words.forEach(word => word.split('').forEach(letter => letters.add(letter)));
+		this.letters = [...letters];
+	}
+
+	edits(word) {
+		const edits = [];
+		const {length} = word;
+		const {letters} = this;
+
+		for (let i = 0; i < length; i++) {
+			edits.push(word.slice(0, i) + word.slice(i + 1)); // Skip
+			for (const letter of letters) {
+				edits.push(word.slice(0, i) + letter + word.slice(i + 1)); // Replace
+			}
+		}
+
+		for (let i = 1; i < length; i++) {
+			edits.push(word.slice(0, i - 1) + word[i] + word[i - 1] + word.slice(i + 1)); // Transposition
+		}
+
+		for (let i = 0; i <= length; i++) {
+			for (const letter of letters) {
+				edits.push(word.slice(0, i) + letter + word.slice(i)); // Addition
+			}
+		}
+
+		return edits;
+	}
+
+	correct(word, distance) {
+		const {words} = this;
+
+		if (words.has(word)) {
+			return word;
+		}
+
+		if (distance > 0) {
+			const edits = this.edits(word);
+
+			for (const edit of edits) {
+				if (words.has(edit)) {
+					return edit;
+				}
+			}
+
+			if (distance > 1) {
+				for (const edit of edits) {
+					const correction = this.correct(edit, distance - 1);
+					if (correction !== undefined) {
+						return correction;
+					}
+				}
+			}
+		}
+	}
+}
+
+const properties = new Set([
+	...util.executionMethods,
+	'context',
+	'title',
+	'skip'
+]);
+
+const correcter = new MicroCorrecter([...properties]);
 
 const isCallExpression = node =>
 	node.parent.type === 'CallExpression' &&
 	node.parent.callee === node;
 
-const getMemberStats = members => {
-	const initial = {
-		skip: [],
-		falsey: [],
-		method: [],
-		other: []
-	};
-
-	return members.reduce((res, member) => {
-		if (member === 'skip') {
-			res.skip.push(member);
-		} else if (member === 'falsey') {
-			res.falsey.push(member);
-		} else if (isMethod(member)) {
-			res.method.push(member);
-		} else {
-			res.other.push(member);
-		}
+const getMemberNodes = node => {
+	if (node.object.type === 'MemberExpression') {
+		return getMemberNodes(node.object).concat(node.property);
+	}
 
-		return res;
-	}, initial);
+	return [node.property];
 };
 
+const correctIdentifier = name => correcter.correct(name, Math.max(0, Math.min(name.length - 2, 2)));
```

Comment:
Not immediately clear what the `2` signifies. Can you move it into a constant or comment?